### ISSUE TITLE
Full Screen

### ISSUE THREAD

**Issue by magadan** [jlesage/docker-firefox#3]
Hello ! Thanks for this container, really cool.

I think FullScreen is not working, Firefox still in the 1280x720  ( HD resolution )...

Is possible to run with entire browser screen ?

**Comment by magadan** [comment]
And... Another question, not relative to this container... 

Do you think is possible to do a Kodi Container ( v17 ) ?

**Comment by jlesage** [comment]
Do you have more details about your full screen issue?

**Comment by bbakermmc** [comment]
@jlesage Is there a way to set the UI to just be 100% of remote?  Currently you need to set a resolution like 10x7, but it doesnt "grow", so do I just set a higher one and have it shrink?

**Comment by jlesage** [comment]
Currently the "remote" resolution doesn't change.  So you need to scale on client side.
However, this may change in the future by changing the vnc server inside the container...

**Comment by bbakermmc** [comment]
@jlesage right, but if I have a 4k monitor, and the docker is set to 10x7, the scale does nothing, I would need to set the resolution to 4k? then on a 1080p monitor it would scale down?

**Comment by jlesage** [comment]
Yes you are right, no upscaling is done, only downscaling.  So setting the resolution to 4k could be a solution.

**Comment by PavelNL** [comment]
The answer is in the original image configuration:

$ docker image inspect jlesage/firefox

            "Env": [
                 ...
                **"DISPLAY_WIDTH=1280",
                "DISPLAY_HEIGHT=768"**

So:
**docker run -d --name=firefox5 -p 5800:5800 --shm-size 2g -e "DISPLAY_WIDTH=1920" -e "DISPLAY_HEIGHT=1080"  jlesage/firefox**

**Comment by ovizii** [comment]
Thanks. very useful info, should be added to your explanations (I mean the part about only downscaling being done :-)
